staging/src/k8s.io/component-base/metrics/opts.go

```diff
@@ -37,6 +37,14 @@ var (
 	allowListLock        sync.RWMutex
 )
 
+// ResetLabelValueAllowLists resets the allow lists for label values.
+// This is useful for testing.
```

Comment:
"useful for testing" or shall we call out that this is explicitly for testing. Using it in prod can result in metric label values that were not allowed earlier to re-appear

Reply:
Updated.